src/Service/Storage/Ef/DbGlobalContext.cs

```diff
@@ -101,6 +102,15 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
                 .IsRequired();
         });
 
+        modelBuilder.Entity<DispatchedEmail>(builder =>
+        {
+            builder.HasKey(x => x.Id);
+            builder.HasOne(x => x.Application)
+                .WithMany(x => x.DispatchedEmails)
+                .HasForeignKey(x => x.Tenant)
+                .IsRequired();
+        });
+
```

Comment:
@Tyrrrz Saw something here, we should probably index on Created since that is how we will query.

Reply:
Done